### PR TITLE
feat(KAMI-234): added new settings module

### PR DESCRIPTION
**Pull request by KamiWize**
## What does this PR do?
Integrates a new module.

## Where should the reviewer start?
``settings.js``

## How should this be manually tested?

1. Checkout to the current branch.
2. npm install & npm start.
3. See the logs.

## What are the relevant tickets?

[KAMI-234](https://wizeline.atlassian.net/browse/KAMI-234)

## Screenshots
Captura de Pantalla 2022-12-21 a la(s) 13 46 47

#### Checklist

- [x] I added the necessary documentation, if appropriate.
- [x] I added tests to prove that my fix is effective or my feature works.
- [x] I reviewed existing Pull Requests before submitting mine.

### CODE REVIEW

File: settings.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log("settings on");
```
Comment (jogeocc) [inline settings.js:1]: OK!

**Review by jogeocc — CHANGES_REQUESTED** [review]
Todo Ok pero falto el archivo .env

File: settings.js (hunk 1 above)
Comment (KamiWize) [reply settings.js:1]: jiji

Comment (jogeocc) [reply settings.js:1]: OK Ya esta

**Review by jogeocc — APPROVED** [review]
OK :)

**Pull request merged**